Source/VMware.vSphereDSC/Classes/InventoryBaseDSC.ps1

```diff
@@ -18,83 +18,188 @@ class InventoryBaseDSC : BaseDSC {
     <#
     .DESCRIPTION
 
+    Name of the Inventory Item from the Datacenter we will use.
+    #>
+    [DscProperty(Key)]
+    [string] $Name
+
+    <#
+    .DESCRIPTION
+
+    The full path to the Inventory Item in the Datacenter we will use from the Inventory.
+    The path consists of 0 or more folders.
+    The Root folders of the Datacenter are not part of the path.
+    The parts of the path are separated with "/".
+    Example path: "<Folder Name>/<Folder Name>"
+    #>
+    [DscProperty(Key)]
+    [string] $InventoryPath
+
+    <#
+    .DESCRIPTION
+
     The full path to the Datacenter we will use from the Inventory.
     The path consists of 0 or more folders and the Datacenter name.
-	Root folder of the Inventory is not part of the path.
+    Root folder of the Inventory is not part of the path.
     The parts of the path are separated with "/" where the last part of the path is the Datacenter name.
     Example path: "<Folder Name>/<Folder Name>/<Datacenter Name>".
     #>
     [DscProperty(Key)]
-    [string] $DatacenterPath
+    [string] $Datacenter
 
     <#
     .DESCRIPTION
 
-    Value indicating if the Inventory object should be Present or Absent.
+    Value indicating if the Inventory Item should be Present or Absent.
     #>
     [DscProperty(Mandatory)]
     [Ensure] $Ensure
 
     <#
     .DESCRIPTION
 
+    The type of Datacenter Folder in which the Inventory Item is located.
```

Comment:
in which the resource is located. Possible values are VM, Network, Datastore, Host